main_hw2.py

```diff
@@ -0,0 +1,96 @@
+def my_sum_function(*args, **kwargs):
+    r_s = 0
+    print(args)
+    print(kwargs)
+    for index, element in enumerate(args):  # print as tuple ; de obicei se foloseste index, element
+        # print(index, element)
+        type_element = type(element).__name__
+        if type_element == 'float' or type_element == 'int':
+            r_s += element
+    for value in kwargs.values():
+        # print(value)
+        type_value = type(value).__name__
+        if type_value == 'float' or type_value == 'int':
+            r_s += value
+
+    return r_s
+
+
+def sum_rec(n):
```

Comment:
Please refactor `sum_rec,` `sum_rec_even` and `sum_rec_odd` as one function.

Reply:
created sum_rec_refact which returns tuple (sum, sum_even) sum_odd is calculated as difference between them; or one can put it in returned tuple as well